findme-users/src/FollowerService.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime
+from typing import List
+
+from pydantic import parse_obj_as
+from src.FollowerRepository import FollowerRepository
+from src.entities.FollowRequest import FollowRequest
+from src.entities.UserConnections import UserConnections
+
+from aws_lambda_powertools.event_handler.exceptions import (
+    BadRequestError,
+)
+
+
+
+class FollowerService:
+    def __init__(self):
+        self.follower_repository = FollowerRepository()
+
+    def create_follower_request(self, requester_id, requestee_id):
+        if requester_id == requestee_id:
+            raise BadRequestError(f"It is not possible to create the follow request since requester ({requester_id}) and requestee ({requestee_id}) are the same person!")
+
+        follow_request = FollowRequest(
+            requester_id=requester_id,
+            requestee_id=requestee_id,
+            status='pending',
+            timestamp=datetime.now()
+        )
+
+        self.follower_repository.create_follow_request(follow_request)
+
+        return follow_request
+
+    def accept_follow_request(self, requester_id, requestee_id):
+        self.follower_repository.accept_follow_request(requester_id, requestee_id)
```

Comment:
we might want to check if the provided user_id exists and if it is not your own

Reply:
it is also possible to accept request for user_id's that do not exist
<img width="885" alt="image" src="https://github.com/uzh-ase-fs24/backend/assets/91185670/c2c02b5e-2f16-4cd8-9b03-97c84c20b03b">